examples/size.rs

```diff
@@ -31,10 +31,11 @@ fn main() {
         println!("      State.backtrace: {}", size_of_val(&state.backtrace));
     }
     #[cfg(not(feature = "backtrace"))]
-    {
+    fn size_of_state() {
         let state = error_chain::State {
             next_error: None,
         };
         println!("      State.next_error: {}", size_of_val(&state.next_error));
     }
+    size_of_state();
```

Comment:
Could you change the code to use `State::default()`? That way, only the second `println` would be compiled conditionally.